index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/1e7e3fccee.js" crossorigin="anonymous"></script>
+  <title>Aly Atalla</title>
+</head>
+<body>
+  <header>
+    <nav class="nav">
+        <div class="menu-button">
+            <i class="fa-solid fa-bars fa-xl"></i>
+        </div>
+        <ul class="header-links">
+            <li class="header-links" href="#works">Experience</li>
+            <li class="header-links" href="#about">About</li>
+            <li class="header-links" href="#contact">Contact</li>
+        </ul>
```

Comment:
- [ ] Good work implementing the required changes here. You did not need to delete the anchor tags. Kindly make the changes like below.
```
<ul>
   <li><a class="header-links" href="#works">Experience</a></li>
   <li><a class="header-links" href="#about">About</a></li>
   <li><a class="header-links" href="#contact-me">Contact</a></li>
</ul>
```